google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -111,10 +114,10 @@ public FluentTrackingEventWithLabel trackEvent(String action) {
     return new TrackingEventBuilder(this, externalPluginName, action);
   }
 
-  private static void sendPing(@NotNull final List<? extends NameValuePair> postData) {
+  private void sendPing(@NotNull final List<? extends NameValuePair> postData) {
```

Comment:
It's to use `userAgent`. Another way could be making `userAgent` static (and possibly other fields). Which one do you think is better?